library/src/main/java/com/github/twocoffeesoneteam/glidetovectoryou/GlideToVectorYou.java

```diff
@@ -50,7 +53,7 @@ public GlideToVectorYou setPlaceHolder(int placeHolderLoading, int placeHolderEr
         return instance;
     }
 
-    public void load(Uri uri, ImageView imageView) {
+    public void load(Uri uri) {
```

Comment:
This will break existing implementations. Maybe add a new method with the method signature that you want?

Reply:
Sure. Kept the older method and just added a new one.